setup.cfg

```diff
@@ -5,24 +5,32 @@ description-file =
     README.rst
 author = Redfish dev team
 author-email = python-redfish@mondorescue.org
-home-page = http://github.com/bcornec/python-redfish
+home-page = https://github.com/uggla/python-redfish/tree/devel
```

Comment:
Hummm. Don't think the devel branch is the one to put upfront. Tagged realease are the one to use. That's also why using your tree for that is not important.